library/cwm/src/lib/cwm/common_widgets.rb

```diff
@@ -194,7 +201,8 @@ class ComboBox < AbstractWidget
     #
     # @param val [Object] Value to assign to the widget
     def value=(val)
-      change_items([[val, val]] + items) if editable? && !items.map(&:first).include?(val)
+      change_items([[val, val]] + current_items) if editable? && !current_items.map(&:first).include?(val)
```

Comment:
This change makes some unit tests to fail. The method `#current_items` returns nil when running unit tests, so this line will fail when it tries to map over nil.

For example, it fails when testing this https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/widgets/fstab_options.rb#L566.